### PR TITLE
added heat map

### PR DESCRIPTION
**Pull request by julieflament**
- heat map shows initial data
- updates based on where user clicks on map
- go to http://127.0.0.1:5000/predictions-dashboard

### CODE REVIEW

**Review by komalsingh0925 — APPROVED** [review]
looks good!

File: server/routing/admin.py [hunk 1]
```diff
@@ -41,20 +41,17 @@ def get_users():
...
             users = []
             for row in rows:
                 user1 = {
                     'id': row[0],
                     'username': row[1],
-                    'password': row[2],
-                    'email': row[3],
-                    'phone': row[4],
-                    'alerts_email': row[5],
-                    'alerts_phone': row[6],
```
Comment (noahstaz) [inline server/routing/admin.py:56]: Wondering why the database structure is changed here?

**Review by noahstaz — CHANGES_REQUESTED** [review]
Most looks good, wondering about the comment I made

**Review by danglakshay — APPROVED** [review]
Looks good! Approved!

**Review by trevorwinser — APPROVED** [review]

File: server/routing/admin.py (hunk 1 above)
Comment (trevorwinser) [reply server/routing/admin.py:56]: > Wondering why the database structure is changed here?

This doesn't change the database structure. The admin dashboard should not show private information such as the email, and phone number of the user.

**Pull request merged**